### PR TITLE
[CDAP-13238] Adds Compute Profiles section to Namespace Details page, and minor bugfixes

### PR DESCRIPTION
**Pull request by tonybach**
JIRA: https://issues.cask.co/browse/CDAP-13238

### CODE REVIEW

File: cdap-ui/app/cdap/components/Cloud/Profiles/ListView/ListView.scss [hunk 1]
```diff
@@ -22,6 +22,22 @@
     max-height: 550px;
     .grid-item {
       grid-template-columns: 20px 1.5fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 20px;
+
+      .sortable-header {
+        cursor: pointer;
+        display: flex;
+        align-items: center;
+        text-align: right;
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/components/Cloud/Profiles/ListView/ListView.scss]: any specific reason why this is align right? I am right now seeing this

screen shot 2018-04-03 at 4 51 12 pm

vs
 
screen shot 2018-04-03 at 4 51 18 pm

File: cdap-ui/app/cdap/components/Cloud/Profiles/ListView/index.js [hunk 2]
```diff
@@ -14,16 +14,90 @@
...
+    label: ''
+  },
+  {
+    property: 'name',
+    label: T.translate(`${PREFIX}.profileName`),
+    defaultSortBy: true
+  },
+  {
+    property: (profile) => (profile.provisioner.name),
+    label: T.translate(`${PREFIX}.provider`)
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/components/Cloud/Profiles/ListView/index.js]: Based on our discussion I think this should be `Provisioner`?

File: cdap-ui/app/cdap/text/text-en.yaml [hunk 3]
```diff
@@ -218,6 +220,22 @@ features:
...
+    Profiles:
+      ListView:
+        assosciations: Assosciations
+        createOne: "creating one"
+        last24HrNodeHr: Last 24 hrs node/hr
+        last24HrRuns: Last 24 hrs runs
+        noProfiles: "No compute profiles have been defined in this namespace. Start by "
+        profileName: Profile Name
+        profileUsage: Profile usage
+        provider: Provider
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/text/text-en.yaml]: nit: Provisioner

**Review by ajainarayanan — COMMENTED** [review]
Talked to @tonybach . Few minor comments.
@tonybach is working on changing the way we render the header for profiles list view to be a callback notification instead of passing on the component as props.

File: cdap-ui/app/cdap/components/Cloud/Profiles/ListView/index.js [hunk 4]
```diff
@@ -14,16 +14,90 @@
...
+const PREFIX = 'features.Cloud.Profiles.ListView';
+
+const PROFILES_TABLE_HEADERS = [
+  {
+    label: ''
+  },
+  {
+    property: 'name',
+    label: T.translate(`${PREFIX}.profileName`),
+    defaultSortBy: true
```
Comment (elfenheart) [inline cdap-ui/app/cdap/components/Cloud/Profiles/ListView/index.js]: what is this `defaultSortBy` being used for?

File: cdap-ui/app/cdap/components/Cloud/Profiles/ListView/index.js [hunk 5]
```diff
@@ -45,62 +119,179 @@ export default class ProfilesListView extends Component {
...
+      return (
+        <div className="text-xs-center">
+          <LoadingSVG />
+        </div>
+      );
+    }
+    if (this.state.error) {
+      return (
+        <div className="text-danger text-xs-center">
+          {JSON.stringify(this.state.error, null, 2)}
```
Comment (elfenheart) [inline cdap-ui/app/cdap/components/Cloud/Profiles/ListView/index.js]: will this render correctly? Stringify-ing the JSON by 2 spaces indentation, and centering the text seems a little odd.

File: cdap-ui/app/cdap/components/Cloud/Profiles/ListView/ListView.scss (hunk 1 above)
Comment (tonybach) [reply cdap-ui/app/cdap/components/Cloud/Profiles/ListView/ListView.scss]: No reason, I didn't look at the mocks correctly

File: cdap-ui/app/cdap/components/Cloud/Profiles/ListView/index.js (hunk 4 above)
Comment (tonybach) [reply cdap-ui/app/cdap/components/Cloud/Profiles/ListView/index.js]: It was used when I was using the `SortableTable` component, but missed removing this when I removed that component.

File: cdap-ui/app/cdap/components/Cloud/Profiles/ListView/index.js (hunk 5 above)
Comment (tonybach) [reply cdap-ui/app/cdap/components/Cloud/Profiles/ListView/index.js]: Good catch 👍

File: cdap-ui/app/cdap/components/NamespaceDetails/ComputeProfiles/ComputeProfiles.scss [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+*/
+
+@import "../../../styles/variables.scss";
+@import "../../../styles/mixins.scss";
+$tr-height: 50px;
+$link-color: #0275d8;
```
Comment (elfenheart) [inline cdap-ui/app/cdap/components/NamespaceDetails/ComputeProfiles/ComputeProfiles.scss]: unused variables and imports?

**Review by elfenheart — APPROVED** [review]
👍 LGTM

**Pull request merged**